src/GraphQL.Tests/Execution/MutationTests.cs

```diff
@@ -362,7 +362,7 @@ public void evaluates_mutations_correctly_in_the_presence_of_a_failed_mutation()
                   ""sixth"": null
                 }";
 
-            var result = AssertQueryWithErrors(query, expected, root: new Root(6, DateTime.Now), expectedErrorCount: 2);
+            var result = await AssertQueryWithErrorsAsync(query, expected, root: new Root(6, DateTime.Now), expectedErrorCount: 2);
```

Comment:
This test hangs/deadlocks within GitHub Actions if not run asynchronously.

Reply:
I saw such an issue in graphql-client repo. XUnit has special constrained (6 workers by defaukt) syncronization context.